index.js

```diff
@@ -71,6 +57,55 @@ module.exports = function(el, options) {
   return tabbableNodes;
 }
 
+/**
+ * Walks the DOM tree starting at a root element and checks if any of its children 
+ * match the provided selectors. Similar to the native `querySelectorAll` except
+ * that it will traverse the shadow DOM as well as slotted nodes.
+ * @param {Element} rootElement The element to start querying from.
+ * @param {string[]} selectors An array of CSS selectors.
+ * @param {boolean} checkRootElement True if the provided root element is to be matched against the selectors.
+ */
+function deepQuerySelectorAll(rootElement, selectors, checkRootElement) {
```

Comment:
Is there an npm package that implements this? It doesn't seem right to me that this library, tabbable, would include its own implementation of such a function. Would be much better to use a shared and thoroughly tested public package.